kivy/uix/textinput.py

```diff
@@ -2252,6 +2252,13 @@ def _get_line_options(self):
 
     def _create_line_label(self, text, hint=False):
         # Create a label from a text, using line options
+        
```

Comment:
what this have to do with the actual issue?

Reply:
Oops! the last two commits are not related to issue. My mistake!

Reply:
Then could you remove this commit if it's not related. Also try to squash your changes so it's only one commit.